nilearn/experimental/surface/_datasets.py

```diff
@@ -116,29 +116,3 @@ def fetch_nki(mesh_type: str = "pial", **kwargs) -> Sequence[SurfaceImage]:
         images.append(img)
 
     return images
-
-
-def fetch_destrieux(
```

Comment:
remove this "shortcut" to make it explicit in examples how we construct the SurfaceImage: [#4642](https://github.com/nilearn/nilearn/issues/4642)